floss.spec

```diff
@@ -124,11 +124,19 @@ a = Analysis(
              ],
              hookspath=None,
              runtime_hooks=None,
-             excludes=None,
+             excludes=["tkinter", "_tkinter", "Tkinter"],
              win_no_prefer_redirects=None,
              win_private_assemblies=None,
              cipher=block_cipher)
 
+a.binaries = a.binaries - TOC([
+ ('sqlite3.dll', None, None),
+ ('tcl85.dll', None, None),
+ ('tk85.dll', None, None),
+ ('_sqlite3', None, None),
+ ('_ssl', None, None),
+ ('_tkinter', None, None)])
+

```

Comment:
not sure which of these two changes actually fixes #140, but i think its the first one. the second one ensures there aren't extra DLLs included as resources.